tests/ui/issues/issues

```diff
@@ -0,0 +1,520 @@
+10291
```

Comment:
What is this file for? Was it added accidentally?

Reply:
Ohh, I thought I ensured it wasn't added. My bad, I will remove it